--- a/etc/init.d/sysfixtime	2018-06-14 04:17:09.000000000 +0200
+++ b/etc/init.d/sysfixtime	2022-07-24 17:58:10.316586625 +0200
@@ -8,10 +8,12 @@
 HWCLOCK=/sbin/hwclock
 
 boot() {
-	start && exit 0
+	start
 
 	local curtime="$(date +%s)"
-	local maxtime="$(find /etc -type f -exec date -r {} +%s \; | sort -nr | head -n1)"
+	[ $curtime -gt "1658670000" ] && exit 0
+
+	local maxtime="$(find /etc -type f -maxdepth 1 -exec date -r {} +%s \; | sort -nr | head -n1)"
 	[ $curtime -lt $maxtime ] && date -s @$maxtime
 }
 
